python/dask_cudf/dask_cudf/expr/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from dask import config
+
+# Check if dask-dataframe is using dask-expr
+QUERY_PLANNING_ON = config.get("dataframe.query-planning", False)
```

Comment:
We need to know at import time if dask-exp is "enabled" (i.e. query planning is "on")